kinode/packages/terminal/pkg/manifest.json

```diff
@@ -7,6 +7,22 @@
         "request_capabilities": [
             "app-store:app-store:sys",
             "contacts:contacts:sys",
+            {
+                "process": "contacts:contacts:sys",
+                "params": "ReadNameOnly"
+            },
+            {
+                "process": "contacts:contacts:sys",
+                "params": "Read"
+            },
+            {
+                "process": "contacts:contacts:sys",
+                "params": "Add"
+            },
+            {
+                "process": "contacts:contacts:sys",
+                "params": "Remove"
+            },
```

Comment:
Rather than requesting these in terminal (special case), let's do the general case thing and *grant* them to terminal:terminal:sys in the manifest for contacts:contacts:sys

Reply:
Agreed

Reply:
Sorry it took so long; changed in 108aacf